ytxp-plutarch/examples/direct-offer/doc/transaction-families/creating.md

```diff
@@ -0,0 +1,77 @@
+# Offer Creating Transaction Family
+
+This transaction family describes the creation of an offer.
+
+## Script Execution Specification
+
+The transactions detailed in this specification are non-composable. This design is due to assumptions regarding the order of outputs. It is defined this way for the sake of performance simplicity.
```

Comment:
I would remove this disclaimer from the tx families. While correct it might make the example a bit more complex to understand. (Same for other tx families).